pkg/plugins/common/kustomize/v2-alpha/scaffolds/api.go

```diff
@@ -95,6 +82,13 @@ func (s *apiScaffolder) Scaffold() error {
 		); err != nil {
 			return fmt.Errorf("error scaffolding kustomize API manifests: %v", err)
 		}
+
+		// If the gvk is non-empty
+		if s.resource.Group != "" || s.resource.Version != "" || s.resource.Kind != "" {
```

Comment:
nits: Just wondering in which case this `gvk` might be empty so that this sample kustomization should be prevented?

Reply:
It is only to prevent somewhere the validation was not made before.
For the kustomize I do not think that we are validating. The check is done in the golang.
So, if be called with another plugin and etc.